src/sentry/api/endpoints/organization_config_integrations.py

```diff
@@ -2,26 +2,19 @@
 
 from rest_framework.response import Response
 
-from django.conf import settings
-
 from sentry import integrations, features
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize, IntegrationProviderSerializer
 
 
 class OrganizationConfigIntegrationsEndpoint(OrganizationEndpoint):
     def get(self, request, organization):
-        has_catchall = features.has(
-            "organizations:internal-catchall", organization, actor=request.user
-        )
         has_pagerduty = features.has("organizations:pagerduty-v2", organization, actor=request.user)
 
         providers = []
         for provider in integrations.all():
             if not has_pagerduty and provider.key == "pagerduty":
                 continue
-            if not has_catchall and provider.key in settings.SENTRY_INTERNAL_INTEGRATIONS:
```

Comment:
@MeredithAnya Was this only ever used for PageDuty? If we do a similar thing with a future integration, say Trello, would it make sense to keep this logic?

Reply:
Idk, I don't feel like we should keep dead code around - it just adds complexity and can be added back fairly easily.

Do we need to keep `settings.SENTRY_INTERNAL_INTEGRATIONS` around?

Reply:
Yeah I think we should remove until we need again - @manuzope good call lemme update

Reply:
Sounds good. We can just revert this commit when we want it back.

Reply:
I would love to see the end of internal_integrations. It doesn't end up saving time as each integration ends up needing their own feature flags.